metabrainz/__init__.py

```diff
@@ -20,6 +20,9 @@ def create_app(debug=None, config_path = None):
         use_flask_uuid=True,
     )
 
+    # get rid of some really pesky warning
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
```

Comment:
maybe better to go in the config file/config file template instead of create_app()?

Reply:
This is one of those annoying messages that makes test runs hard to read -- I dont want to clutter the config files more than they already are. I'll make a note to remove this in a year, when hopefully it isn't needed anymore.